taskweaver/code_interpreter/code_generator/code_generator.py

```diff
@@ -57,13 +56,17 @@ def __init__(
         plugin_registry: PluginRegistry,
         logger: TelemetryLogger,
         llm_api: LLMApi,
-        code_verification_config: CodeVerificationConfig,
         round_compressor: RoundCompressor,
+        plugin_only: bool = False,
+        code_verification_on: bool = False,
```

Comment:
wondering why we need to have these configurations in constructor when we also have a `configure_verification` method

Reply:
removed